cmd/platform/deploy_test.go

```diff
@@ -144,6 +145,20 @@ func TestDeployCommand_HasValidDeploymentMethod(t *testing.T) {
 			manifestSource: config.ManifestSourceLocal,
 			expectedError:  slackerror.New(slackerror.ErrSDKHookNotFound),
 		},
+		"succeeds if the app exists and the manifest source is remote": {
```

Comment:
😉 thought: These are clever tests. I think it calls out the importance of error codes in assertions. But seems right here!